### PR TITLE
Rename UserTask to WorkBatch

### PR DESCRIPTION
**Pull request by withrocks**
We're renaming the UserTask model to WorkBatch. This change was mostly automatic and I then ran migrations for the renamed models.

### CODE REVIEW

File: src/sentry/api/endpoints/project_tags.py [hunk 1]
```diff
@@ -35,3 +35,31 @@ def get(self, request, project):
             )
 
         return Response(data)
+
+
+# TODO: Move to clims
+from sentry.api.bases.organization import OrganizationEndpoint
+
+
+class OrganizationWorkBatchPropertiesEndpoint(OrganizationEndpoint):
```
Comment (johandahlberg) [inline src/sentry/api/endpoints/project_tags.py]: Is this related to the rename?

Comment (withrocks) [reply src/sentry/api/endpoints/project_tags.py]: No, thanks fixed it!
